src/Phansible/RoleManager.php

```diff
@@ -50,7 +50,7 @@ public function getInitialValues()
      */
     public function setupRole(array $requestVars, VagrantBundle $vagrantBundle)
     {
-        array_walk($this->roles, function(Role $role) use($requestVars, $vagrantBundle) {
+        array_walk($this->roles, function (Role $role) use ($requestVars, $vagrantBundle) {
```

Comment:
It follows (PSR-2 closues](https://github.com/php-fig/fig-standards/blob/master/accepted/PSR-2-coding-style-guide.md#6-closures) style guide.